packages/forms-web-app/package.json

```diff
@@ -57,6 +57,7 @@
 		"pino": "^6.13.2",
 		"proj4": "^2.9.0",
 		"redis": "^4.5.1",
+		"sequelize": "6.33.0",
```

Comment:
might not be needed. forms-web-app doesn't use it as far as i know

Reply:
Was also added automatically but yes I cant see it on this package so not sure why it was added - will remove manually